permissions/src/main/java/com/google/accompanist/permissions/MultiplePermissionsState.kt

```diff
@@ -36,10 +36,12 @@ import androidx.compose.ui.util.fastMap
 @Composable
 public fun rememberMultiplePermissionsState(
     permissions: List<String>,
-    onPermissionsResult: (Map<String, Boolean>) -> Unit = {}
+    onPermissionsResult: (Map<String, Boolean>) -> Unit = {},
+    permissionStatuses: Map<String, PermissionStatus> = emptyMap()
```

Comment:
If this would prevent a release because of the API change I can move it to a separate PR. Not sure what the policy is with default params, but `metalavaGenerateSignatureRelease` seems to treat it as an API change (also see next commit where I address this with `JvmOverloads`).

Reply:
Yeah, default params don't fix binary compat unfortunately. Just add a new function and then make this one call your new one and then it is fine

Reply:
Does the `JvmOverloads` work for that, or does there have to be an actual new function in source?

Reply:
Good question, I'm not 100% sure actually but on Compose we always do it with a new function in source so I assume there is a reason

Reply:
Ok I'll push that shortly.